java/android/nnstreamer/src/androidTest/java/org/nnsuite/nnstreamer/APITestCommon.java

```diff
@@ -42,6 +99,9 @@ public static void initNNStreamer() {
                 fail();
             }
         }
+
+        /* Copy nnstreamer folder in assets to the app's files directory */
+        copyAssetFolder(getContext(), "nnstreamer", getContext().getFilesDir().getAbsolutePath() + "/nnstreamer/");
```

Comment:
suggestion:
add static method APITestCommon.getRootDirectory()
```
initNNStreamer() {
...
  mootDirectory = context.getFilesDir().getAbsolutePath();
...


public static String getRootDirectory() {
  return mRootDirectory;
}
```

Reply:
PTAL for the code updated. I follow your suggestion